src/mkdocstrings_handlers/python/handler.py

```diff
@@ -128,6 +149,7 @@ class PythonHandler(BaseHandler):
             If Black is installed, the signature is also formatted using it. Default: `False`.
 
     Attributes: Additional options:
+        show_function_title (bool): Whether to display the title of the function/class method object. Default: `True`.
```

Comment:
Or maybe `show_function_title` option is more suitably listed under the

```
Attributes: Headings options:
```
section up above.

Reply:
As said in Conversation tab, I'd like to drop this from this PR for now.

Reply:
removed